src/deepak/assignment49_FileHandling/FreqOfWordInEntireFile.java

```diff
@@ -0,0 +1,53 @@
+package deepak.assignment49_FileHandling;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class FreqOfWordInEntireFile {
+	
+	String searchString = "Technocredits";
+	int count;
+	
+	void readFile() {
+		
+		
+		try {
+			
+			Scanner sc = new Scanner(new File("G:\\Technocredits\\Project\\Java_Program\\Eclipse_Practice\\javatechnooct2020\\src\\deepak\\assignment49_FileHandling\\Sample.txt"));
```

Comment:
you should give relative path of file, this code will not work on others system.